.github/workflows/ci.yml

```
@@ -238,11 +238,23 @@ jobs:
           checks/requirements.py
 
   notebook:
-    name: Notebook check
     env:
-      SUPERSTAQ_API_KEY : ${{ secrets.SUPERSTAQ_API_KEY }}
-      IBMQ_TOKEN : ${{ secrets.IBMQ_TOKEN }}
+      SUPERSTAQ_API_KEY: ${{ secrets.SUPERSTAQ_API_KEY }}
+      IBMQ_TOKEN: ${{ secrets.IBMQ_TOKEN }}
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - type: "qiskit-superstaq"
+            pattern: "*_qss.ipynb"
+            dependencies: "-e ./checks-superstaq -e ./general-superstaq[dev] -e ./qiskit-superstaq[dev]"
+          - type: "cirq-superstaq"
+            pattern: "*_css.ipynb"
+            dependencies: "-e ./checks-superstaq -e ./general-superstaq[dev] -e ./cirq-superstaq[dev]"
+          - type: "other docs"
+            pattern: "--exclude=**/*_css.ipynb --exclude=**/*_qss.ipynb"
```

assuming prior (+ cleanup)
```suggestion
            pattern: "--exclude *_css.ipynb *_qss.ipynb 'cirq-superstaq/*' 'qiskit-superstaq/*'"
```

i think all this leaves is `ibmq_dd_strategies` and the HPCA tutorial though. The former could probably just be renamed `ibmq_dd_strategies_qss` (it doesn't use css at all), and the latter should maybe be split up into qss and css versions? (definitely outside the scope of this pr though)